### ISSUE TITLE
Advice on creating a first-class Webjar package

### ISSUE THREAD

**Issue by bcopy** [webjars/webjars#1653]
Hello,

My project (blockly-pygamezero) is a Webjar package made from the ground-up - there is no upstream distribution (it's a Javascript code generator library for Pygamezero).
Would you consider forking it to the webjars github organization ? 

https://github.com/chiby-hq/blockly-pygamezero

Can you advise on which groupId, metadata I should include ? Just the same as a regular "upstream-based" package ?

**Comment by jamesward** [comment]
Cool!  Are you interested in having it in the webjars org so we will handle the releases to Maven Central?  Or so it shows up on webjars.org?  Something else?

**Comment by bcopy** [comment]
Yes being in maven webjars.org and on the website. So I just need to
produce a webjar and change the groupid ?

**Comment by jamesward** [comment]
Well, it is tricky because webjars.org pulls in artifacts with the `org.webjars` groupId from Maven Central.  But only we can publish artifacts with that groupId.  So to make that change we would have to fork it into the webjars org (so it is in a place we control).  We can definitely do that but you'll lose some ability to control releases.  Let me know if you want to move forward that way.

**Comment by bcopy** [comment]
Hi James,

Yes fine by me, releases shouldn't be too frequent, I'll just file merge
requests. The repo is mentioned above, kindly let me know what to change on
my side to minimize the merging.

**Comment by jamesward** [comment]
Cool.  Here is the fork: https://github.com/webjars/blockly-pygamezero

Let me know if you'd like me to perform a first release.

**Comment by bcopy** [comment]
Thanks - I'll update and let you know
